keps/sig-apps/4368-support-managed-by-for-batch-jobs/README.md

```diff
@@ -278,7 +278,7 @@ a typo in the field value, or the cluster administrator may not install the
 custom controller (like MultiKueue) on that cluster.
 
 In such cases the user may not observe any progress by the job for a long time
-and my need to debug the Job.
+and may need to debug the Job.
 
 In order to allow for debugging of situations like this the Job controller will
```

Comment:
A few lines below, there's a statement about expanding kubectl to print information by which controller the job is managed. Was there a decision one way or the other?